test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt

```diff
@@ -274,3 +273,9 @@ fun String.normalizeFilePath(): String =
     } catch (e: Throwable) {
         this
     }
+
+fun List<String>.normalizeToTestTargets(): ShardChunks {
+    return if (this.isEmpty().not()) {
+        this.map { it.split(',', ';') }
+    } else emptyList()
+}
```

Comment:
Less code to read == less code to understand
```suggestion
fun List<String>.normalizeToTestTargets(): ShardChunks =
    if (isEmpty()) emptyList()
    else map { it.split(',', ';') }
```

Reply:
Already resolved.